server/routes/nl_interface.py

```diff
@@ -46,12 +46,15 @@ def _get_preferred_type(types):
 
 
 def _highlight_svs(sv_df):
+  if sv_df.empty:
+    return []
   return sv_df[sv_df['CosineScore'] > 0.4]['SV'].values.tolist()
 
 
-def _filtered_svs_list(sv_df):
-  sv_df = sv_df.drop(sv_df[sv_df['CosineScore'] < 0.3].index)
-  return sv_df['SV'].values.tolist()
+def _filtered_svs_df(sv_df):
+  if sv_df.empty:
+    return _empty_svs_score_dict()
```

Comment:
should we return a dataframe here since the non empty path returns a dataframe? We probably don't want one function to return two different types

Reply:
Ah great catch! Done